scripts/install_ebpf.psm1

```diff
@@ -219,48 +161,179 @@ function Install-eBPFComponents
           [parameter(Mandatory=$true)] [string] $KmTraceType,
           [parameter(Mandatory=$false)] [bool] $KMDFVerifier = $false)
 
-    # Stop eBPF Components
-    Stop-eBPFComponents
+    # Print the status of the eBPF drivers and services before installation.
+    # This is useful for detecting issues with the runner baselines.
+    Print-eBPFComponentsStatus "Querying the status of eBPF drivers and services before the installation (none should be present)..." | Out-Null
+
+    # Install the Visual C++ Redistributable (Release version, which is required for the MSI installation).
+    Write-Log("Installing Visual C++ Redistributable from '$VcRedistPath'...")
+    $process = Start-Process -FilePath $VcRedistPath -ArgumentList "/quiet", "/norestart" -Wait -PassThru
+    if ($process.ExitCode -ne 0) {
+        Write-Log("Visual C++ Redistributable installation FAILED. Exit code: $($process.ExitCode).") -ForegroundColor Red
+        throw ("Visual C++ Redistributable installation FAILED. Exit code: $($process.ExitCode).")
+    }
+    Write-Log("Cleaning up...")
+    Remove-Item $VcRedistPath -Force
+    Write-Log("Visual C++ Redistributable installation completed successfully!") -ForegroundColor Green
+
+    # Copy the VC debug runtime DLLs to the system32 directory,
+    # so that debug versions of the MSI can be installed (i.e., export_program_info.exe will not fail).
+    Write-Log("Copying VC debug runtime DLLs to the $system32Path directory...")
+    # Test if the VC debug runtime DLLs are present in the working directory (indicating a debug build).
+    $VCDebugRuntime = $VCDebugRuntime | Where-Object { Test-Path (Join-Path $WorkingDirectory $_) }
```

Comment:
Umm... MS does not officially distribute the 'debug' dlls.  Are we breaking that rule here?

Reply:
This step is just copying the debug DLLs already present in the downloaded build artifacts.
@shankarseal fyi

Reply:
@shankarseal: Should our build artifacts even contain these debug dlls?

I'd like to know the official guidance here, specifically in relation to the following paragraph (the last one) from https://learn.microsoft.com/en-us/cpp/windows/determining-which-dlls-to-redistribute?view=msvc-170:

"You can't redistribute all of the files that are included in Visual Studio. You're only permitted to redistribute the files that are specified in Redist.txt or the online "REDIST list." **_You also can't redistribute debug versions of your applications, or the debug versions of DLLs or libraries supplied by Microsoft_** (italics mine). You may only copy debug executables and libraries to machines you use for testing."

Reply:
> @shankarseal: Should our build artifacts even contain these debug dlls?
> 
> I'd like to know the official guidance here, specifically in relation to the following paragraph (the last one) from https://learn.microsoft.com/en-us/cpp/windows/determining-which-dlls-to-redistribute?view=msvc-170:
> 
> "You can't redistribute all of the files that are included in Visual Studio. You're only permitted to redistribute the files that are specified in Redist.txt or the online "REDIST list." **_You also can't redistribute debug versions of your applications, or the debug versions of DLLs or libraries supplied by Microsoft_** (italics mine). You may only copy debug executables and libraries to machines you use for testing."

The last line does say that we're allowed to copy them over to the test machine, so I guess we're good.  I'm guessing we're getting them from the VS installation on the machine where these artifacts were generated.

Resolving this one.